engines/scumm/string.cpp

```diff
@@ -652,9 +659,9 @@ void ScummEngine::fakeBidiString(byte *ltext, bool ignoreVerb) const {
 		if (_game.id == GID_INDY4 && ltext[0] == 0x7F) {
 			ltext[start + ipos + ll] = 0x80;
 			ltext[start + ipos + ll + 1] = 0;
-		} else if (_game.id == GID_MONKEY2 && ltext[0] == 0x07) {
-			ltext[0] = ' ';
-			ltext[start + ipos + ll] = 0x07;
+		} else if (_game.id == GID_MONKEY2) {
+			// add non-printable character to end to avoid space trimming
+			ltext[start + ipos + ll] = '@';
```

Comment:
Is this needed even when the string is not manipulated?

Reply:
It doesn't seem to do anything on other cases, and it is hard to detect if the string was manipulated (unless adding some flag variable)

Reply:
Ok